aldryn_events/models.py

```diff
@@ -438,33 +472,15 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
-class EventListPlugin(BaseEventPlugin):
-    STYLE_CHOICES = [
-        (STANDARD, _('Standard')),
-    ]
+class EventCalendarPlugin(BaseEventPlugin):
 
-    style = models.CharField(
-        verbose_name=_('Style'),
-        choices=STYLE_CHOICES + get_additional_styles(),
-        default=STANDARD,
-        max_length=50
+    cache_duration = models.PositiveSmallIntegerField(
+        default=0,  # not the most sensible, but consistent with older versions
+        blank=False,
+        help_text=_(
+            "The maximum duration (in seconds) that this plugin's content "
+            "should be cached.")
     )
```

Comment:
NOTE: When I attempted to use a ModelMixin for this package, Django kept trying to tell me that I couldn't use @python_2_unicode_compatible, because the class didn't declare __str__(), which, of course it does.